simplehound/core.py

```diff
@@ -12,8 +12,14 @@
 BAD_API_KEY = 401
 
 ## API urls
-URL_DETECTIONS_BASE = "https://{}.sighthoundapi.com/v1/detections"
+URL_DETECTIONS_BASE = "https://{}.sighthoundapi.com/v1/{}"
 ALLOWED_MODES = ["dev", "prod"]
+ALLOWED_ENDPOINTS = ["detections","recognition"]
+
+RECOGNITION_PARAMS = (
+    ("objectType", "licenseplate"),
```

Comment:
Could run but vehicle and licenseplate recognition here. However, the response JSON is structured differently than a single call to licenseplate or vehicle parameters.